---
title: "Metabolic Effects of PCSK9 Inhibition With Evolocumab in Subjects with Elevated Lp(a)"
output: word_document
bibliography: /home/xiang/Documents/Bibliography/lpa.bib
csl: journal-of-lipid-research.csl
---

Xiang Zhang, PhD$^1,^2$,
Lotte C.A. Stiekema, MD$^2$,
Erik S.G. Stroes, PhD$^2$,
Albert K. Groen, PhD$^1$

1. Department of Experimental Vascular Medicine, Amsterdam University Medical Center, University of Amsterdam, Meibergdreef 9, 1105 AZ Amsterdam, The Netherlands
2. Human and Animal Physiology, Wageningen University, De Elst 1, 6708 WD Wageningen, The Netherlands
3. Department of Vascular Medicine, Amsterdam University Medical Center, University of Amsterdam, Meibergdreef 9, 1105 AZ Amsterdam, The Netherlands

# Abstract

Background: 
Epidemiological studies substantiated that subjects with elevated lipoprotein(a) [Lp(a)] have a markedly increased cardiovascular risk. 
Inhibition of proprotein convertase subtilisin/kexin type 9 (PCSK9) lowers both LDL cholesterol (LDL-C) as well as Lp(a), albeit modestly. 
Effects of PCSK9 inhibition on circulating metabolites such as lipoprotein subclasses, amino acids and fatty acids remain to be characterized. 

Methods:
We performed nuclear magnetic resonance (NMR) metabolomics on plasma samples derived from 30 individuals (placebo N = 14; evolocumab N = 16). 
We assessed the effects of evolocumab treatment on circulating metabolites by running lognormal regression analyses. 
Subsequently, we assessed the interrelationship between Lp(a) and 14 lipoprotein subclasses in response to treatment of evolocumab, by running multilevel multivariate regression analyses. 

Results:
On average, evolocumab treatment for 16 weeks resulted in a 17% (95% credible interval: 8% to 26%) reduction of circulating Lp(a), coupled with substantial reduction of VLDL, IDL and LDL particles as well as their lipid contents. 
Although we did not observe correlations between Lp(a) lowering and reduction in other 14 lipoprotein subclasses, we identified that increasing concentrations of baseline Lp(a) were associated with larger reduction in triglyceride-rich VLDL particles after evolocumab treatment.  

Conclusions:
Inhibition of PCSK9 with evolocumab markedly reduced VLDL particle concentrations in addition to lowering LDL-C. 
The substantial reduction in VLDLs after evolocumab treatment indicates enhanced clearance of VLDL particles in patients with high Lp(a). 

# Introduction

Lipoprotein(a) [Lp(a)], synthesized in the liver, is a low-density lipoprotein (LDL)-like particle covalently bound to the apolipoprotein(a) [apo(a)]. 
Lp(a) is cleared from the circulation primarily by the liver [@Cain2005], and the exact mechanism remains to be elucidated. 
Epidemiological studies substantiated a markedly increased cardiovascular risk in subjects with elevated Lp(a), involving both pro-inflammatory as well as pro-coagulant effects [@ODonoghue2019]. 
To date, limited therapeutic options are available to reduce Lp(a) in plasma [@VanCapelleveen2016]. 
One agent that lowers Lp(a) levels is proprotein convertase subtilisin/kexin type 9 (PCSK9) monoclonal antibody, resulting in increased abundance of LDL receptors [@Raal2014; @Raal2016].
The excess number of LDL receptors on the surface of hepatocytes may contribute to enhanced Lp(a) catabolism [@Romagnuolo2015; @Raal2016]. 
However, this concept is not supported the effect of statin therapy, which also increases hepatic LDL receptor abundances with a concomitant increase of plasma Lp(a) [@Tsimikas2019]. 
Hence, the exact role of the LDL-receptor in Lp(a) catabolism has not been elucidated and remains a matter of debate [@Reyes-Soffer2017; @Watts2018; @Stoekenbroek2018]. 

Besides the LDL receptor, PCSK9 also induces the degradation of the VLDL (very low density lipoprotein) receptor, ApoER2 (apolipoprotein E receptor 2), and CD36 (cluster of differentiation 36) [@Poirier2008; @Demers2015]. 
Interestingly, the VLDL receptor and CD36 are receptors for triglyceride-rich lipoproteins (TRLs) as well as Lp(a), indicating a potential interaction between Lp(a) and TRL metabolism [@Argraves1997; @Seimon2010; @Dijk2018]. 
A link between Lp(a) and TRL metabolism is supported by a recent NMR (nuclear magnetic resonance) metabolomics study, in which a Lp(a)-raising genotype was found to associate with decreasing concentrations of triglyceride-rich VLDL particles [@Kettunen2016]. 
The mechanism underlying this inverse relationship is unknown. 
Another NMR metabolomics study showed that genetic inhibition of PCSK9 and statin therapy had similar effects on circulating metabolites [@Sliz2018]. 
However, genetic inhibition of PCSK9 differs from inhibition with monoclonal antibody, since genetic inhibition includes intracellular functions of PCSK9. 
Moreover, data on the relationship between Lp(a) lowering and metabolism of other lipoproteins are scarce.  

In the present study, we analyzed NMR metabolomics of plasma derived from 30 subjects with elevated Lp(a), using either placebo (n = 14) or PCSK9 inhibitor (evolocumab, n = 16). 

# Methods

## Study design

This was a substudy of the ANITSCHKOW trial (NCT02729025) conducted at the Amsterdam UMC in The Netherlands between April 2016 and July 2017. 
The ANITSCHKOW trial was a phase 3b, multicenter, randomized, double-blind, placebo-controlled trial using subcutaneous injections of evolocumab 420mg Q4W for 16 weeks as investigational product [@Stiekema2018].
The study was conducted in accordance to the principles of the Declaration of Helsinki and the study protocol was approved by the ethic committee of the Amsterdam UMC. 

Patients were found to be eligible for enrollment in the ANITSCHKOW trial when they were $\geq$ 50 years of age, had a fasting LDL-C of $\geq$ 2.6 mmol/L (100mg/dL), and an Lp(a) of $\geq$ 125nmol/L (50mg/dL). 
At the time of randomization, study subjects who were on lipid lowering treatment had to be on a stable dosing regimen for at least 8 weeks. 
Exclusion criteria were among others, diabetes mellitus and a cardiovascular event within three months prior to randomization. 
The complete list of eligibility criteria was reported previously [@Stiekema2018].

## Biochemical measurements

Blood samples were obtained at time of randomization and 16 weeks after treatment. Patients were fasting for $\geq$ 9 hours for both blood withdrawals. 
Total cholesterol, high-density lipoprotein (HDL) cholesterol, triglycerides, and apolipoproteinB-100 (ApoB-100) were measured by commercially available kits at the Medpace core lab (Medpace Reference Laboratories, Leuven, Belgium). 
Low-density lipoprotein (LDL) cholesterol was calculated using the Friedewald formula. 
Ultracentrifugation-determined LDL-C was measured and reported if calculated LDL cholesterol was below 40mg/dL, or triglycerides were above 400mg/dL. 
Lipoprotein(a) levels were measured using an isoform-independent immunoturbidometric assay (Polymedco, Cortlandt Manor, NY, USA) and reported in nmol/L. 

## Metabolite quantification

Quantification of 225 metabolic measures was performed by using a high-throughput NMR metabolomics platform (Nightingale health, Finland) [@Soininen2009]. 
The 225 metabolic measures contain around 150 primary concentrations as well as ratios that were derived from the primary concentrations. 
In this study, we focused on primary concentrations of circulating metabolites that cover multiple metabolic pathways including lipoproteins, fatty acids as well as amino acids and glycolysis intermediates. 
The following 14 lipoprotein subclasses and their lipid compositions were quantified: 
extremely large (average particle diameter >75 nm), 
very large (average particle diameter 64.0 nm), large (53.6 nm), medium (44.5 nm), small (36.8 nm) and very small VLDL (31.3 nm); 
intermediate density lipoprotein (IDL; 28.6 nm); 
three LDL subclasses, i.e. large (25.5 nm), medium (23.0 nm) and small LDL (18.7 nm); 
and four HDL subclasses, i.e. very large (14.3 nm), large (12.1 nm), medium (10.9 nm) and small HDL (8.7 nm).
The complete list of the 225 metabolic measures can be found at  https://nightingalehealth.com/biomarkers

## Statistical analysis

### Metabolic effects of PCSK9 inhibition with evolocumab

To assess the effect of evolocumab on levels of a circulating metabolite, we ran lognormal regression analysis. 
The outcome variable ($y$) was the concentration of a metabolite. 
Two predictor variables were in the regression model: 1) evolocumab treatment (evolocumab, $T_i$ = 1; placebo $T_i = 0$); 
2) metabolite concentration at time of randomization ($x$).
The baseline concentrations were centered and scaled so that the mean is 0 and standard deviation is 1.
The lognormal distribution was chosen to model the outcome variable because its values were positive continuous. 
Due to missing observations in outcome variables, we applied the Bayesian approach to handle missing data. 
There were two types of missing values: 
(1) when the concentration of a metabolite is below the limit of detection, 
or (2) when values were rejected by the automatic sample and measurement quality control procedure in the Nightingale pipeline. 
All the missing observations were assumed missing at random and treated as parameters.
Values were randomly drawn from a lognormal distribution. 
Regarding the missing values that were below the limit of detection, the imputed values were constrained between zero and the minimal observed value. 
We fitted the model by running Hamiltonian Markov Chain Monte Carlo in the program Stan (version 2.18.3).
We ran four Markov chains with 2000 iterations in each chain. 
Results were presented with the posterior mean with 95% credible interval (CI). 

### Multilevel multivariate model
    
To assess the relationship between Lp(a) lowering and metabolism of 14 lipoprotein subclasses, we developed a multilevel multivariate model. 
The detailed model is available at https://github.com/XiangZhangSC/Anitschkow.
In short, 16 subjects from the treatment group were used for this analysis, and each subject had two plasma samples at time of randomization and after treatment.   
The outcome variable is a vector of 2, $[L_i, y_i]$, in which $L_i$ represents the Lp(a) concentration in sample $i$, and $y_i$ represents the particle concentration of a lipoprotein subclass in sample $i$. 
The logarithm of $[L_i, y_i]$ was modeled by a multivariate normal distribution, with parameters $\mu_L$ (a vector of 32), $\mu_y$ (a vector of 32) and $\Sigma$ (a 2-by-2 covariance matrix). 
$\mu_L[i]$ represents the mean concentration of Lp(a) in sample $i$, and was modeled as 
$$\mu_L[i] = \alpha_{L,subject[i]} + \beta_{L,subject[i]} \times V_2[i]$$ 
$\alpha_L$ is a vector of 16, and represents the Lp(a) abundance at baseline. 
$\beta_L$ is a vector of 16, and represents the effect of evolocumab treatment on Lp(a). 
$V_2$ is a vector of 32 with 0s and 1s. 
When $V_2[i] = 1$, it means that measurement was derived from the second visit (after treatment). 
Otherwise, $V_2[i] = 0$. 
Similarly, $\mu_y[i]$ represents the mean concentration of a lipoprotein subclass in sample $i$, and was modeled as 
$$\mu_y[i] = \alpha_{y,subject[i]} + \beta_{y,subject[i]} \times V_2[i]$$ 
$\alpha_y$ is a vector of 16, and represents the lipoprotein subclass abundance at baseline. 
$\beta_y$ is a vector of 16, and represents the effect of evolocumab treatment on the lipoprotein subclass. 
To assess the relationship between the Lp(a) lowering and the metabolism of a lipoprotein subclass, we used another multivariate normal distribution to model the subject-specific parameters. 
$$[\alpha_L, \beta_L, \alpha_y, \beta_y][j] \sim MultivariateNormal([\theta_L, \gamma_L, \theta_y, \gamma_y], \Sigma_{subj})$$

$\theta_L$, $\gamma_L$, $\theta_y$, $\gamma_y$ and $\Sigma_{subj}$ (a 4-by-4 covariance matrix) are the hyper-parameters. 
We fitted the model by running Hamiltonian Markov Chain Monte Carlo in the program Stan (version 2.18.3).
We ran four Markov chains with 2000 iterations in each chain. 
Results were presented with the posterior mean with 95% credible interval (CI). 

# Results

The current study included 30 patients (evolocumab, n = 16; placebo, n = 14). 
Baseline characteristics of the two groups were comparable and are provided in Table 1. 

## Metabolic effects of PCSK9 inhibition with evolocumab

On average, evolocumab treatment for 16 weeks resulted in a 17% (95% credible interval: 8% to 26%) reduction in Lp(a), together with a concomitant 67% (57%, 76%) and 21% (6% 35%) reduction in LDL cholesterol and triglyceride, respectively. 

To identify the metabolic effects corresponding to PCSK9 inhibition with evolocumab, we performed NMR metabolomics covering metabolic pathways such as lipoprotein subclasses, fatty acids, amino acids and glycolysis. 
We observed that evolocumab treatment resulted in substantial reduction in particle concentration of extremely large (80% [48% 100%]), very large (90% [70% 100%]), large (60% [34%, 83%]), medium (50% [36%, 63%]), small (39% [32% 46%]) and very small VLDL (47% [40% 53%]).
We also observed that evolocumab treatment resulted in particle concentration reduction in IDL (53% [45%, 60%]), large (56% [48% 65%]), medium (59% [50%, 67%]) and small LDL (55% [47%, 64%]). 
In addition, we observed that evolocumab treatment resulted in decreased concentrations of very large HDL particles (24% [3% 45%]), and increased concentrations of medium HDL particles (13% [4% 23%]) (Figure 1).

![Figure 1 Mean difference between evolocumab and palacebo group, adjusting for pre-treatment lipoprotein particle concentrations. Circles represent the posterior means. Lines refer to the 95% credible intervals](./Treatment_effect_lipoproteins.png)

Similar to the lipoprotein particle concentration profiles, we observed that evolocumab treatment resulted in substantial reduction of esterified cholesterol (CE), triglyceride (TG), free cholesterol (FC) and phospholipid (PL) in VLDL, IDL and LDL as well as very large HDL (Figure 2). 

![Figure 2 Mean difference in lipoprotein lipid compositions between evolocumab and placebo group, adjusting for pre-treatment lipoprotein lipid concentrations. Circles represent the posterior means. Lines refer to the 95% credible intervals. CE: esterified cholesterol; FC: free cholesterol; PL: phospholipids; TG: triglyceride.](./Treatment_effect_lipoprotein_lipid.png)


The NMR metabolomics also quantified the fatty acid content in the lipoproteins. 
We observed that treatment of evolocumab resulted in 30% [24% 36%] reduction in total fatty acids, with the largest effect on docosahexaenoic acid (DHA 40% [24% 55%]) (Figure 3).

![Figure 3 Mean difference in fatty acid concentrations between evolocumab and placebo group, adjusting for pre-treatment fatty acid concentrations. Circles represent the posterior means. Lines refer to the 95% credible intervals.](./Treatment_effect_FFAs.png)

We observed no difference in concentrations of other metabolites including amino acids, fluid balance, glycolysis and ketone bodies. 

## Relationship between Lp(a) lowering and reduction in lipoprotein subclasses

To identify the relationship between Lp(a) lowering and reduction in lipoprotein subclasses, we developed a multilevel multivariate model. 
We observed that Lp(a) lowering was not associated with reduction of the 14 lipoprotein subclasses (Figure 4). 

![Figure 4 Relationship between Lp(a) lowering and reduction in 14 lipoprotein subclasses. Circles represent the posterior mean Pearson correlation coefficients. Lines refer to the 95% credible intervals.](./corr_lipoproteins_lpa.png)

However, we identified that the reduction in medium-sized VLDL particles was associated with increasing baseline Lp(a) concentrations (Pearson correlation coefficient -0.5 [-0.8, -0.06]). 
The correlations between baseline Lp(a) concentrations and reduction in other VLDL particles, including extremely large, very large, large, small and very small VLDLs, were -0.1 [-0.6, 0.3], -0.03 [-0.4, 0.4], -0.3 [-0.7, 0.2], -0.1 [-0.7, 0.6], and 0.2 [-0.3, 0.7], respectively. 

![Figure 5 Reduction of medium VLDL particles correlated with baseline lipoprotein(a) concentrations. Every circle represents the posterior mean reduction of medium VLDL particle concentration and the posterior mean of baseline lipoprotein(a) in a patient treated with evolocumab. The verticle and horizontal bar represents the 95% credible interval. The blue dashed line represented the average percentage (50%) change in medium VLDL particle after evolocumab treatment.](./Baseline_lpa_mVLDL.png)

# Discussion

The major finding of this work is that evolocumab treatment in patients with high Lp(a) massively decreases VLDLs in addition to the well known effect on LDLs. 
In the current work, evolocumab treatment resulted in on average 17% reduction in plasma Lp(a) concentration, together with 67% and 21% reduction in LDL cholesterol and triglyceride. 
These treatment effects were similar to the outcomes reported in the past, indicating that the subjects involved in the current study were representative for the entire ANITSCHKOW trail [@Stiekema2018]. 

Our detailed metabolomic profiling revealed that PCSK9 inhibition with evolocumab resulted in a substantial reduction in concentrations of VLDL, IDL and LDL particles as well as their lipid and fatty acid contents. 
We did not observe changes in other circulating metabolites such as amino acids and ketone bodies. 
In general, PCSK9 inhibition with evolocumab resulted in similar metabolic changes as genetic inhibition of *PCSK9* and statin therapy [@Wurtz2016; @Sliz2018]. 
However, in contrast to genetic inhibition of *PCSK9* and statin therapy that had much larger effects on LDL particles than VLDL particles, we observed that evolocumab treatment induced a more substantial reduction in VLDL particles compared to LDL particles. 
Our multivariate analysis showed that Lp(a) lowering did not correlate with reduction in any of the 14 lipoprotein subclasses, suggesting alternative pathways in Lp(a) catabolism, such as scavenger receptor class B type I (SR-BI) [@Yang2013] and plasminogen receptor [@Sharma2017]. 

Interestingly, we identified that subjects with higher baseline Lp(a) showed a larger reduction in medium VLDL particle concentrations after treatment with evolocumab. 
We observed that subjects with higher baseline Lp(a) also showed tendency to have a larger reduction in extremely large and large VLDL particle concentrations after treatment with evolocumab. 
However, those tendencies had large uncertainty due to considerable number of missing observations. 
The percentage of missing observations in extremely large and large were 69% and 47%, respectively. 
Similarly, since the very large VLDL particle had the largest percentage of missing data (72%) among all the 14 lipoprotein subclasses, we simply had not enough data to detect the relationship between its reduction and baseline Lp(a).  
In contrast, the percentage of missing observations in medium VLDL was 9%, allowing us to assess the relationship between its reduction and baseline Lp(a) levels. 
On the other hand, we did not observe similar relationship between baseline Lp(a) and reduction in small and very small VLDL particles, although the percentages of missing observations were small for these two lipoprotein subclasses (6% and 3%), suggesting that Lp(a) may not influence catabolism of these two triglyceride-poor VLDL particles.    
Our observation is in line with the association between the Lp(a)-raising genotype and decreasing concentrations of extremely large, very large, large and medium VLDL particles, but not small and very small VLDL particles [@Kettunen2016]. 
Lp(a) particles can bind noncovalently to triglyceride-rich lipoproteins (TRL), forming a Lp(a)-TRL complex which was suggested to facilitate receptor-mediated uptake [@McConathy1994; @Gaubatz2001]. 
This process may be particularly important in our subjects with high Lp(a), since they may have a large amount of the Lp(a)-TRL complex. 
Together with the supraphysiological abundance of lipoprotein receptors induced by evolocumab treatment, these two factors may underlie the massive reduction of VLDL particles. 
On the other hand, apo(a) can be recycled to the extracellular space and bind to apoB-containing particles again, resulting in moderate reduction of Lp(a) [@Sharma2017]. 
We find that our hypothesis has connections to other studies. 
First, in the ANITSCHKOW trail, evolocumab had no impact on arterial wall inflammation in patients with persistent Lp(a) elevation [@Stiekema2018]. 
This could be caused by the fact that inhibition of PCSK9 may increase other receptors such as VLDL receptors and CD36. 
Both the VLDL receptor and CD36 are present in macrophages and endothelial cells. 
The accelerated uptake of a Lp(a)-TRL complex via these receptors would not attenuate foam cell formation. 
Second, when Lp(a) is directly targeted via antisense oligonucleotide treatment, reduction of Lp(a) production does decrease cellular inflammatory responses [@Viney2016]. 
In this case the reduced Lp(a) production may decrease the abundance of Lp(a)-TRL complex and uptake of TRL by macrophages, thereby reducing foam cell formation. 
Future studies are required in order to test our hypothesis. 

## Strengths and limitations

Our study provided the first NMR metabolomics data for a PCSK9 inhibition trial. 
The detailed metabolomic profiling not only allowed us to reveal systemic effects of PCSK9 inhibition, but also to assess the relationship between Lp(a) lowering and 14 lipoprotein subclasses. 
We did not detect statistically significant correlations between Lp(a) lowering and reduction in lipoprotein subclasses probably due to the small number of participants and considerable missing data in some lipoprotein subclasses. 
We feel it is worth to apply the multilevel multivariate model developed in this study in a larger clinical trial to improve our understanding of interrelationship between Lp(a) lowering and metabolism of other lipoprotein subclasses. 

In conclusion, our NMR metabolomic profiling revealed that evolocumab treatment in patients with high Lp(a) massively decreases VLDLs in addition to the well known effect on LDLs. 
The substantial reduction in VLDLs after evolocumab treatment suggested enhanced clearance of VLDL particles in patients with high Lp(a). 

# Authors

# Acknowledgements

# Sources of Funding

# Disclosures

# Affiliations

# References

